test/test_manylinuxXXXX_only.py

```diff
@@ -64,10 +70,14 @@ def test(manylinux_image, tmp_path):
         add_env['CIBW_SKIP'] = 'pp*'
     elif manylinux_image in {'manylinux2014', 'manylinux_2_24'}:
         # We don't have a manylinux2014 / 'manylinux_2_24' image for PyPy (yet?)
-        add_env['CIBW_SKIP'] = 'cp27* pp*'  # Python 2.7 not available on manylinux2014 / 'manylinux_2_24'
+        add_env[
+            'CIBW_SKIP'
+        ] = 'cp27* pp*'  # Python 2.7 not available on manylinux2014 / 'manylinux_2_24'
```

Comment:
It doesn't really work here anyway, since it's no more or less relevant to this line than the PyPy comment above. Same line comments should be avoided most of the time.